.github/actions/setup-build-env/action.yml

```diff
@@ -0,0 +1,53 @@
+name: "Setup Dependencies Variables"
+description: "Setup Dependencies Variables from other files"
+outputs:
+  CATTLE_KDM_BRANCH:
+    description: "CATTLE_KDM_BRANCH from package/Dockerfile"
+    value: ${{ steps.vars.outputs.CATTLE_KDM_BRANCH }} 
+  CATTLE_K3S_VERSION: 
+    description: "CATTLE_K3S_VERSION from package/Dockerfile"
+    value: ${{ steps.vars.outputs.CATTLE_K3S_VERSION }}
+  CATTLE_HELM_VERSION: 
+    description: "CATTLE_HELM_VERSION from package/Dockerfile"
+    value: ${{ steps.vars.outputs.CATTLE_HELM_VERSION }}
+  HELM_VERSION: 
+    description: "HELM_VERSION from package/Dockerfile"
+    value: ${{ steps.vars.outputs.HELM_VERSION }}
+  HELM_UNITTEST_VERSION: 
+    description: "HELM_UNITTEST_VERSION"
+    value: 0.3.2
```

Comment:
Same comment as made by Sergey. We should move this value to some config file, or currently source it from: https://github.com/rancher/rancher/blob/c17b50f03dbd4b911de2411ca797a53b9565e659/Dockerfile.dapper#L11